CHANGELOG.md

```diff
@@ -52,6 +52,8 @@ These changes are available on the `master` branch, but have not yet been releas
   ([#2611](https://github.com/Pycord-Development/pycord/pull/2611))
 - Fixed `TypeError` when passing `skus` parameter in `Client.entitlements()`.
   ([#2627](https://github.com/Pycord-Development/pycord/issues/2627))
+- Fixed type annotations for attributes of `context`
```

Comment:
This is not only used by context, maybe something more general like this could fit better.
```suggestion
+ Fixed type annotations of cached properties
```